Cleanup (#164)
configcat/docs: website/docs/sdk-reference/dotnet.md

### JSON File

> The SDK can be set up to load your feature flag & setting overrides from a file. 

Reviewer: ```suggestion
The SDK can load your feature flag & setting overrides from a file. 
```

```

## Using ConfigCat behind a proxy

> Provide your own network credentials (username/password), and proxy server settings (proxy server/port) by injecting a HttpClientHandler instance into the ConfigCatClient's initialization call.

Reviewer: ```suggestion
Provide your own network credentials (username/password) and proxy server settings (proxy server/port) by injecting a HttpClientHandler instance into the ConfigCatClient's initialization call.
```

When the ConfigCat SDK does not work as expected in your application, please check for the following potential problems:

* **Symptom:** Instead of the actual value, the default one is constantly returned by `GetValue()`/`GetValueAsync()` and
  the log contains the following message (provided that the client is set up to log error level events as described [here](#logging)):
  "Secure connection could not be established. Please make sure that your application is enabled to use TLS 1.2+."

>   **Problem:** ConfigCat CDN servers are set up to require TLS 1.2 or newer security protocol for communication.

Reviewer: ```suggestion
  **Problem:** ConfigCat CDN servers require TLS 1.2 or newer security protocol for communication.
```

  As for allowed security protocols, please keep in mind that newer .NET runtimes rely on operating system settings,

>   older versions, however, may need additional set up to make secure communication with the CDN servers work.

Reviewer: ```suggestion
  older versions, however, may need additional setup to make secure communication with the CDN servers work.
```

https://grammarist.com/spelling/set-up-vs-setup/
